index.html

```diff
@@ -7,15 +7,15 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet"/>
   <link href="https://fonts.googleapis.com/css?family=Roboto+Mono" rel="stylesheet"/>
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"/>
-  <link href="/static/app.css" rel="stylesheet"/>
-  <link href="/static/favicon.ico" rel="icon" type="image/x-icon"/>
+  <link href="/app.css" rel="stylesheet"/>
+  <link href="/favicon.ico" rel="icon" type="image/x-icon"/>
 </head>
```

Comment:
could we keep these as `/static`? we need to load these from the `/static` subdirectory (or some prefix) in the drone server binary, so would be nice to use the same path in drone, and in the drone-ui server